README.md

```diff
@@ -1,3 +1,28 @@
-# scrapy_capstone
-# scrapy_capstone
-# scrapy_capstone
+#Parse GridDynamics Blog with Scrapy and visualize it
```

Comment:
Please describe steps to setup environment and how to run your code

Reply:
Steps added to the file READme